hera_mc/tests/__init__.py

```diff
@@ -15,7 +15,7 @@
 import pytest
 
 from hera_mc.utils import get_iterable
-TEST_DEFAULT_REDIS_HOST = 'redishost'
+TEST_DEFAULT_REDIS_HOST = 'localhost'
```

Comment:
did you mean to commit this?

Reply:
I changed the host back, but forgot to commit and push.  I couldn't help myself to fix the duplicated code in cminfo_redis_snap...